observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,107 @@ The Network Observability Operator enables administrators to observe and analyze
+* Previously, when enabling the metric `workload_flows_total` instead of `namespace_flows_total`, the health dashboard stopped showing `By namespace` flow charts. With this fix, the health dashboard now shows these charts by querying the `workload_flows_total` metric. (link:https://issues.redhat.com/browse/NETOBSERV-1746[*NETOBSERV-1746*])
+* Previously, when using the `FlowMetrics` API to generate a custom metric and later modifying its labels,such as adding a new label, the metric stopped populating, and an error was shown in the `flowlogs-pipeline` logs. With this fix, it is now possible to modify the labels, and the error is no longer raised in the `flowlogs-pipeline` logs. (link:https://issues.redhat.com/browse/NETOBSERV-1748[*NETOBSERV-1748*])
+* Previously, there was an inconsistency with the default Loki `WriteBatchSize` configuration: it was set to 100KB in the `FlowCollector` CRD default, and 10MB in the OLM sample or default configuration. Both are now aligned to 10MB, which generally provides better performances and less resource footprint. (link:https://issues.redhat.com/browse/NETOBSERV-1766[*NETOBSERV-1766*])
+* Previously, eBPF flow filter on ports without specifying a protocol were ignored. With this fix, you can set eBPF flow filters independently on ports and or protocols. (link:https://issues.redhat.com/browse/NETOBSERV-1779[*NETOBSERV-1779*])
+* Previously, traffic from Pods to Services was hidden from the *Topology view*; only the return traffic from Services to Pods was visible, allowing to display Services. With this fix, that traffic is correctly displayed. (link:https://issues.redhat.com/browse/NETOBSERV-1788[*NETOBSERV-1788*])
+* Previously, for non cluster administrator users that had access to Network Observability, an error was shown in the console plugin when they tried to filter for something that triggered auto-completion, such as a namespace. With this fix, no error is displayed, and the auto-completion returns the expected results. (link:https://issues.redhat.com/browse/NETOBSERV-1798[*NETOBSERV-1798*])
+* When the secondary interface support was added, there was a need to iterate multiple times to register the per network namespace with the netlink to learn about interface notifications. At the same time, unsuccessful handlers caused a leaking file descriptor because with TCX hook, unlike TC, handlers needed to be explicitly removed when the interface went down. Furthermore, when the network namespace was deleted, there was no go close channel event to terminate the netlinl goroutine socket, which caused go threads to leak. Now, there is no longer leaking file descriptors or go threads when creating and deleting pods. (link:https://issues.redhat.com/browse/NETOBSERV-1805[*NETOBSERV-1805*])
```

Comment:
```suggestion
* When the secondary interface support was added, you had to iterate multiple times to register the per network namespace with the netlink to learn about interface notifications. At the same time, unsuccessful handlers caused a leaking file descriptor because with TCX hook, unlike TC, handlers needed to be explicitly removed when the interface went down. Furthermore, when the network namespace was deleted, there was no go close channel event to terminate the netlinl goroutine socket, which caused go threads to leak. Now, there are no longer leaking file descriptors or go threads when you create or delete pods. (link:https://issues.redhat.com/browse/NETOBSERV-1805[*NETOBSERV-1805*])
```
I'm not sure whether some of the terminology here (`go close channel`, `netlinl goroutine socket`) is correctly spelled or capitalized.

Reply:
netlink is the interfaces events monitoring package used in the netobserv agent, and its a lowercase. 
netinl should be netlink. 
i think goroutine can be lowercase, afaict in searching google, but Go can be capitalized, since its a language.